files/class-vip-filesystem-local-stream-wrapper.php

```diff
@@ -1129,47 +1136,45 @@ private function backtrace_fmt() {
 	/**
 	 * Add a file to the list of files that should be handled locally
 	 *
-	 * @param string $file_path Path to the file
+	 * @param string $file_path Path to the file or a pattern.
 	 * @return bool True if the file was added, false otherwise
 	 */
 	public static function add_local_file( $file_path ) {
 		if ( empty( $file_path ) || ! is_string( $file_path ) ) {
 			return false;
 		}
 
-		if ( ! in_array( $file_path, static::$local_files, true ) ) {
-			static::$local_files[] = $file_path;
-			return true;
+		// Check if pattern contains wildcards (* or ? or [).
+		$is_pattern = str_contains( $file_path, '*' ) || str_contains( $file_path, '?' ) ||
```

Comment:
```suggestion
		$is_pattern = strpbrk( $file_path, '*?[' ) !== false;
```
This would be faster I think

Reply:
Neat! TIL